static/templates/settings/profile_field_choice.hbs

```diff
@@ -3,7 +3,7 @@
     <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
     <input type='text' placeholder='{{t "New option" }}' value="{{ text }}" />
 
-    <button type='button' class="button rounded small btn-danger delete-choice" title="{{t 'Delete' }}">
+    <button type='button' class="button rounded small delete-choice" title="{{t 'Delete' }}">
```

Comment:
Why `btn-danger` class is removed? I think we should keep it to make all delete buttons consistent.

Reply:
This was discussed here [CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/Make.20.22Add.20and.20Edit.20custom.20profile.20field.22.20a.20modal.20.2321634)

Reply:
Ok, thanks.